app/services/github_service.rb

```diff
@@ -18,8 +18,20 @@ def create_new_account_pull_request(account_name, account_description, programme
     master = @client.commit(github_repo, 'master')
     create_branch github_repo, new_branch_name, master.sha
 
-    accounts_path = 'terraform/accounts.tf'
-    accounts_contents = @client.contents github_repo, path: accounts_path
+    for accounts_path in ['terraform/accounts.tf.json', 'terraform/accounts.tf'] do
+      begin
```

Comment:
This could potentially be written as:
```
['terraform/accounts.tf.json', 'terraform/accounts.tf'].each do |account_path|
```

Reply:
I'm relying on `accounts_path` still being in scope after the loop in this case.